datalake_api/querier.py

```diff
@@ -216,10 +216,13 @@ def _exclude_outside(self, records, start, end):
         return [r for r in records if self._is_between(r, start, end)]
 
     def _is_between(self, record, start, end):
-        if record['metadata']['end'] is not None and \
-           record['metadata']['end'] < start:
-            return False
-        if record['metadata']['start'] > end:
+        m = record['metadata']
+        if 'end' not in m or m['end'] is None:
+            if m['start'] < start or m['start'] > end:
+                return False
+            else:
+                return True
+        if m['end'] < start or m['start'] > end:
             return False
         return True
```

Comment:
Is the intention for `_is_between` to check if the start and end in the record are _both_ between the given start and end times, or just that the record includes all/part of the given range?  (It would be helpful to have comments in the code here.)